metadata.yaml

```diff
@@ -44,6 +44,10 @@ requires:
   alertmanager:
     interface: alertmanager_dispatch
 
+peers:
+  replicas:
```

Comment:
I don't think we use this anywhere. And definitely we said we should not add a peer relation to handle this use-case.

Reply:
Not intended to.
Are there any undesired implications for having it declared?

Reply:
The rescale does _not_ need a peer relation, because the relation unit databags will be cleaned up by Juju automatically and that is the issue that is making us redesign the relation, which is preferable than having the `loki_push_api` depend on a `peer` relation for reasons of encapsulation.

Reply:
@Abuelodelanada per comments above I think we can drop this for now.